graphql/user/fields.ts

```diff
@@ -210,6 +210,18 @@ export class UserFieldsResolver {
         return await getAppointmentsForUser(user, take, skip, cursor, direction);
     }
 
+    @FieldResolver((returns) => [Achievement])
```

Comment:
If it's supposed to return only one achievement, it should be reflected in the annotaion.
```suggestion
    @FieldResolver((returns) => Achievement)
```